CONTRIBUTING.md

```diff
@@ -127,14 +124,19 @@ git commit --amend
 git push origin my-feature-branch -f
 ```
 
-#### Check on Your Pull Request
+## Check on Your Pull Request
 
-Go back to your pull request after a few minutes and see whether it passed muster with Travis-CI. Everything should look green, otherwise fix issues and amend your commit as described above.
+Go back to your pull request after a few minutes and see whether it passed
+muster with Travis-CI. Everything should look green, otherwise fix issues and
+amend your commit as described above.
 
-#### Be Patient
+## Be Patient
 
-It's likely that your change will not be merged and that the nitpicky maintainers will ask you to do more, or fix seemingly benign problems. Hang on there!
+It's likely that your change will not be merged and that the nitpicky
+maintainers will ask you to do more, or fix seemingly benign problems. Hang on
```

Comment:
`s/Hang on/Hang in/` (unless `Hang on` is some local slang I'm unware of)

Reply:
No, you are totally right, and this document was originally written a little bit casually, so that's the style.